build/images/base/Dockerfile

```diff
@@ -37,10 +37,67 @@ RUN set -eux; \
     wget -q -O - https://github.com/containernetworking/plugins/releases/download/$CNI_BINARIES_VERSION/cni-plugins-linux-${pluginsArch}-$CNI_BINARIES_VERSION.tgz | tar xz -C /opt/cni/bin $CNI_PLUGINS; \
     wget -q -O /opt/cni/bin/whereabouts https://github.com/k8snetworkplumbingwg/whereabouts/releases/download/$WHEREABOUTS_VERSION/whereabouts-${pluginsArch} && chmod +x /opt/cni/bin/whereabouts
 
-FROM antrea/openvswitch:${BUILD_TAG}
+FROM ubuntu:22.04 as suricata-builder
 
 ARG SURICATA_VERSION
 
+ENV SURICATA_DEPS build-essential \
+                  pkg-config \
+                  libpcre3-dev \
+                  libpcap-dev \
+                  libcap-ng-dev \
+                  libnspr4-dev \
+                  libyaml-dev \
+                  libmagic-dev \
+                  libjansson-dev \
+                  libevent-dev \
+                  libnfnetlink-dev \
+                  libnetfilter-queue-dev \
+                  libnetfilter-log-dev \
+                  libnet1-dev \
+                  liblz4-dev \
+                  libnss3-dev \
+                  python3-dev \
+                  zlib1g-dev \
+                  rustc cargo
+
+ENV SURICATA_RUNTIME_DEPS libpcre3 \
+                          libpcap0.8 \
+                          libcap-ng0 \
+                          libnspr4 \
+                          libyaml-0-2 \
+                          libmagic1 \
+                          libjansson4 \
+                          libevent-2.1-7 \
+                          libevent-pthreads-2.1-7 \
+                          libnfnetlink0 \
+                          libnetfilter-queue1 \
+                          libnetfilter-log1 \
+                          libnet1 \
+                          liblz4-1 \
+                          libnss3 \
+                          python3 \
+                          python3-simplejson \
+                          zlib1g
+
+RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates curl $SURICATA_DEPS $SURICATA_RUNTIME_DEPS && \
+    curl -OL https://www.openinfosecfoundation.org/download/suricata-${SURICATA_VERSION}.tar.gz && \
+    tar zxf suricata-${SURICATA_VERSION}.tar.gz && \
+    cd suricata-${SURICATA_VERSION} && \
+    ./configure --prefix=/usr --sysconfdir=/etc --localstatedir=/var --disable-shared --disable-gccmarch-native --enable-nfqueue && \
```

Comment:
Why we add `--disable-gccmarch-native` here?

Reply:
`--disable-gccmarch-native` is recommended when the binary can run on different hardware (including VMs).
It is also used in the official Ubuntu build (which produces the binaries for the OISF PPA) and in this [Dockerfile](https://github.com/jasonish/docker-suricata/blob/main/6.0/Dockerfile.amd64) from one of the Suricata maintainers, which I used as a starting point.

I can add a comment

Reply:
Thanks for the explanation. I understand now.